modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -2052,3 +2052,11 @@
 
 /obj/item/storage/box/large/custom_kit/trickingtrapster/PopulateContents()
 	new /obj/item/tarotdeck(src)
+
+// Time, blood and sweat donor?
+/datum/gear/donator/kits/crimsonquiver
```

Comment:
For custom loadouts, our policy is to have them be on par with roundstart weapons. For this, it would need to be a much weaker version of the gun here.

Reply:
Dang that sorta defeats the point of an iconic pistol

Reply:
but if you still think it should be nerfed i could notch down the damage by like a tier or something, since i had to switch to damage modifiers instead of bullet damage cause people at discord dev talk said i should not make a new caliber and reuse one existant.